### PR TITLE
Fix stale ignore: prefer_const_constructors from flutter/packages

### PR DESCRIPTION
**Pull request by ayinloya**
Update the flutter version in affected packages to 3.10.0
Fix stale ignore: prefer_const_constructors from flutter/packages

Fixes https://github.com/flutter/flutter/issues/128141

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Review by gspencergoog — APPROVED** [review]

**Review by domesticmouse — APPROVED** [review]

**Comment by stuartmorgan** [comment]
Overriding version check: The changes that are being flagged are:
- example main.dart files, but having that updated on pub.dev isn't important enough to publish just for that, and
- upping minimum SDK versions, which we deliberately don't publish for without other changes per [the exemption list](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version)

File: packages/flutter_adaptive_scaffold/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## NEXT
+
+* Fixes stale ignore: prefer_const_constructors
```
Comment (stuartmorgan) [inline packages/flutter_adaptive_scaffold/CHANGELOG.md:3]: This is missing the period at the end (applies to all the changelogs)

File: packages/url_launcher/url_launcher/example/ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -169,7 +169,7 @@
 		97C146E61CF9000F007C117D /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastUpgradeCheck = 1300;
+				LastUpgradeCheck = 1430;
```
Comment (stuartmorgan) [inline packages/url_launcher/url_launcher/example/ios/Runner.xcodeproj/project.pbxproj:172]: Please revert this change, as it is unrelated.

File: packages/url_launcher/url_launcher/example/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1300"
+   LastUpgradeVersion = "1430"
```
Comment (stuartmorgan) [inline packages/url_launcher/url_launcher/example/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme:3]: Same here.

File: script/configs/windows_unit_tests_exceptions.yaml [hunk 4]
```diff
@@ -15,6 +15,7 @@
 - camera_web
 - file_selector
 - file_selector_web
+- google_maps_flutter
```
Comment (stuartmorgan) [inline script/configs/windows_unit_tests_exceptions.yaml:18]: And this (which has now landed on `main`).

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
Thanks for cleaning this up! Just a few minor nits, then it'll be good to land.

**Pull request merged**